neurom/fst/_neuronfunc.py

```diff
@@ -33,7 +33,12 @@
 from neurom.core.types import NeuriteType
 from neurom.core.types import tree_type_checker as is_type
 from neurom.core.dataformat import COLS
-from neurom import morphmath as mm
+from neurom import morphmath
+
+
+def neurons_pop(nrns):
```

Comment:
Yeah, I wasn't a fan either.  Neurons_as_iterable is too generic, as it could mean any of the many iters that neurons have, in my mind.

How about `neuron_population`?